db/migrations/00023_create_headers_table.sql

```diff
@@ -8,7 +8,8 @@ CREATE TABLE public.headers
     block_timestamp      NUMERIC,
     check_count          INTEGER NOT NULL DEFAULT 0,
     eth_node_id          INTEGER NOT NULL REFERENCES eth_nodes (id) ON DELETE CASCADE,
-    eth_node_fingerprint VARCHAR(128)
+    eth_node_fingerprint VARCHAR(128),
+    UNIQUE (block_number, hash, eth_node_fingerprint)
```

Comment:
I'm always super anxious about us half-supporting syncing with several clients, since I'm not certain we actually account for that everywhere ._.

Reply:
Yeah, agreed. Really what this is accomplishing, though, is just making sure that running against multiple chains (e.g. mainnet/kovan) won't cause destructive operations against one another. Definitely would like to do more to make multi-client support for one chain robust